filebeat/config/config.go

```diff
@@ -26,9 +33,9 @@ type Config struct {
 
 type FilebeatConfig struct {
 	Prospectors  []*common.Config `config:"prospectors"`
-	SpoolSize    uint64           `config:"spool_size"`
+	SpoolSize    uint64           `config:"spool_size" validate:"min=1"`
 	PublishAsync bool             `config:"publish_async"`
-	IdleTimeout  time.Duration    `config:"idle_timeout"`
+	IdleTimeout  time.Duration    `config:"idle_timeout" validate:"nonzero,min=0"`
```

Comment:
that's right. if not setting any unit, the default is 's'. But normally I set units.